helm/crossplane/templates/extra-objects.yaml

```diff
@@ -0,0 +1,4 @@
+{{ range .Values.extraObjects }}
```

Comment:
Interesting concept to have this in a Helm chart 🤔

Reply:
agreed. I think this was a last minute addition that they published on the release notes as "a feature"